packs/jira/aliases/create_issue.yaml

```diff
@@ -0,0 +1,18 @@
+---
+name: "create_jira_issue"
+action_ref: "jira.create_issue"
+description: "Create a JIRA issue."
+formats:
+    - "create issue with summary {{summary}} description {{description}} of type {{type}}"
+    - "JIRA summary {{summary}} description {{description}} type {{type}} fix_version {{fix_version}}"
+    - "JIRA summary {{summary}} description {{description}} type {{type}} fix_version {{fix_version}} assignee {{assignee}}"
+    - "JIRA summary {{summary}} description {{description}} type {{type}} fix_version {{fix_version}} assignee {{assignee}} labels {{labels}}"
+    - "JIRA summary {{summary}} description {{description}} type {{type}} fix_version {{fix_version}} assignee {{assignee}} labels {{labels}} components {{components}}"
+    - "JIRA summary {{summary}} description {{description}} type {{type}} fix_version {{fix_version}} assignee {{assignee}} labels {{labels}} components {{components}} priority {{priority}}"
```

Comment:
That waterfall of formats looks pretty crazy :cactus: We're agile now!

Doesn't make it somehow easier, comparing to using Web.
Probably for real life usage 1 shortest format is enough.